package.json

```
@@ -252,7 +252,8 @@
     "sucrase@npm:3.34.0": "^3.35.0",
     "@expo/config/glob": "^10.3.10",
     "@expo/config-plugins/glob": "^10.3.10",
-    "@metamask/network-controller": "patch:@metamask/network-controller@npm%3A18.1.2#~/.yarn/patches/@metamask-network-controller-npm-18.1.2-1bcb8d8610.patch"
+    "@metamask/network-controller": "patch:@metamask/network-controller@npm%3A18.1.2#~/.yarn/patches/@metamask-network-controller-npm-18.1.2-1bcb8d8610.patch",
+    "rpc-websockets": "^8.0.1"
```

Nit: This would also prevent us from updating this package beyond this version, anywhere in the codebase. We can minimize the impact of this resolution by scoping it:
```suggestion
    "@solana/web3.js/rpc-websockets": "^8.0.1"
```